src/styles/_person.scss

```
@@ -0,0 +1,37 @@
+.person {
+  text-align: center;
+  &__photo {
+    width: 150px;
+    height: 150px;
+    border-radius: 50%;
+  }
+
+  &__name {
+    margin-top: 12px;
+    font-weight: bold;
+  }
+
+  &__position {
+    margin: 0;
+    color: #696973;
+    font-style: italic;
+  }
+
+  &__text {
+    position: relative;
+
+    &::after {
+      content: "";
+      display: block;
+      position: absolute;
+      width: 40px;
+      height: 33px;
+      background-image: url("../images/quotes.svg");
+      background-size: contain;
+      background-repeat: no-repeat;
+      background-position: center;
+      right: calc(100% + 6px);
```

одно и тоже ведь, только читается проще
```suggestion
      left: -6px;
```